Homework 4

mmefremov opened the pull request:
test: add checkFilterByRating, checkFilterByRatingOnMapOverlay in BookingTest and methods for its Pages.
refactor: change type CHECK_IN, CHECK_OUT to LocalDate.
style: add a method for each field data in OrderPage, rename variables.
fix: add method to close onetrust banner.

mmefremov committed:
test: add checkFilterByRating, checkFilterByRatingOnMapOverlay in BookingTest and methods for its Pages.
refactor: change type CHECK_IN, CHECK_OUT to LocalDate.
style: add a method for each field data in OrderPage, rename variables.
fix: add method to close onetrust banner.

as-besedin commented on src/test/java/pages/booking/MapFullOverlay.java:
```diff
@@ -0,0 +1,28 @@
+import static com.codeborne.selenide.Condition.*;
+import static com.codeborne.selenide.Selenide.$;
+
+public class MapFullOverlay {
```
класс должен кончаться на Page

mmefremov committed:
maven: add dependencies for allure-report.
test: add test and page for MTS.
allure: add annotations, resources properties.
style: add on-demand static import for Assert, Duration.

mmefremov committed:
maven: add dependencies for lombok, jackson-datatype.
test: add model, endpoint and tests for API.
